### PR TITLE
Update README

### LINKED ISSUE #1: README is not crystal clear
The current version of the README omits important information that may be necessary for less experienced developers willing to run things locally. For example, it does not mention the need for a pre-existing Bandada group or a `supabase` account to run everything locally.

### PR DESCRIPTION
**Pull request by 0xjei**
This PR updates the README to reflect whatever is needed to run the Feedback app demo locally. 

__Minor__
update the lock file to resolve conflicts.

closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,64 +1,111 @@
...
-https://bandada-semaphore-demo.vercel.app/
+The Feedback Application is already deployed and configured with a manual off-chain group and a credential off-chain group.
 
-Video demo: 
-https://youtu.be/ji6E5ES8JcI
+You can begin interacting with both immediately without any additional configuration. Alternatively, you can watch a step-by-step walkthrough of how to use each application.
 
-### Using off-chain credential groups:
+- Manual off-chain group ([demo](https://bandada-semaphore-demo.vercel.app/), [video](https://youtu.be/ji6E5ES8JcI))
+- Credential off-chain group ([demo](https://credential-semaphore-demo.vercel.app/), [video](https://youtu.be/K_rAzjDPYyI))
```
Comment (vplasencia) [inline README.md]:
```suggestion
- Credential off-chain group ([demo](https://credential-bandada-semaphore.vercel.app/), [video](https://youtu.be/K_rAzjDPYyI))
```

File: README.md [hunk 2]
```diff
@@ -1,64 +1,111 @@
...
+Copy the environment variables for the development environment, run this command:
 
 ```bash
-cp env.development.local.example .env.development.local
+cp .env.development.local.example .env.development.local
 ```
 
-and add your environment variables: group id and group api key (if the group is not a credential group).
+```bash
+# These can be retrieved in the Bandada dashboard (e.g., https://<dashboard_url>/groups/off-chain/<group_id>).
```
Comment (vplasencia) [inline README.md]:
```suggestion
# These can be retrieved from the Bandada dashboard (e.g., https://<dashboard_url>/groups/off-chain/<group_id>).
```

**Review by vplasencia — CHANGES_REQUESTED** [review]
Hey @0xjei 

Thanks! Very well explained. Great work! 🔥

**Comment by vplasencia** [comment]
@0xjei why do you have a new `yarn.lock` file? 🤔

File: README.md [hunk 3]
```diff
@@ -1,64 +1,111 @@
...
+
+## Use credential off-chain group locally
+
+**Please make sure you are on the `credential-group-app` branch.**
+
+You must meet the same prerequisites and follow the same steps as previously outlined for the manual off-chain group but instead you have to create a credential off-chain group.
+
+**nb.** This tutorial assumes you are using Github Followers (_MinFollowers_) as credential provider.
+
+You need to **[Register a new OAuth application](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app)** using your Github account. Then configure the `GITHUB_CLIENT_ID` and `GITHUB_CLIENT_SECRET` environment variables in the [Bandada APIs](https://github.com/privacy-scaling-explorations/bandada/blob/main/apps/api/.env.example) with the values that Github gives you after setting up the OAuth application. Note that this is necessary for Bandada to be able to act as a credential provider and validator for your group.
```
Comment (vplasencia) [inline README.md]: Did you test this part? Because I remember that GitHub needed `https` to check everything, then I didn't test these things locally.

Comment (0xjei) [reply README.md]: This should work fine. We can leave as is right now or flag as experimental. What do you prefer?

**Comment by 0xjei** [comment]
@vplasencia ready for review

**Review by vplasencia — APPROVED** [review]

**Pull request merged**